examples/profile.js

```diff
@@ -25,22 +25,57 @@ app.MainController = function($http, $scope) {
     this['profileData'] = resp.data['profile'];
   }));
 
-  this['point'] = null;
-};
-
-
-/**
- * @param {Object} point The point object. It correspond to an item in the list
- * of points given to the profile.
- */
-app.MainController.prototype.onProfileHover = function(point) {
-  this['point'] = point;
-};
-
-
-/**
- */
-app.MainController.prototype.onProfileOut = function() {
+  /**
+   * @param {Object} item
+   * @return {number}
+   */
+  var z = function(item) {
+    return item['values']['mnt'];
+  };
+
+  /**
+    * @param {Object} item
+    * @return {number}
+    */
+  var dist = function(item) {
+    return item['dist'];
+  };
+
+  /**
+   * @type {ngeox.profile.ProfileExtractor}
+   */
+  var extractor = {z: z, dist: dist};
+
+
+  // Using closures for hoverCallback and outCallback since
+  // wrapping in angular.bind leads to a closure error.
+  var that = this;
+
+  /**
+   * @param {Object} point
+   */
+  var hoverCallback = function(point) {
+    // An item in the list of points given to the profile.
+    that['point'] = point;
+  };
+
+  var outCallback = function() {
+    that['point'] = null;
+  };
+
+
+  /**
+   * @type {ngeox.profile.ProfileOptions}
+   */
+  var profileOptions = {
+    extractor: extractor,
+    hoverCallback: hoverCallback,
+    light: false, // explicitely set to avoid closure error
```

Comment:
Once again, can't we use a default value? Does it really have to be explicitely set?